x-pack/plugins/security_solution/public/flyout/document_details/right/components/header_title.tsx

```diff
@@ -102,25 +111,41 @@ export const HeaderTitle: FC = memo(() => {
         {isAlert && !isEmpty(ruleName) ? ruleTitle : eventTitle}
       </div>
       <EuiSpacer size="m" />
-      <EuiFlexGroup direction="row" gutterSize="m" responsive={false}>
-        {isAlert && !isPreview && (
-          <EuiFlexItem grow={false}>
-            <DocumentStatus />
-          </EuiFlexItem>
-        )}
-        <EuiFlexItem grow={false}>
-          <RiskScore />
-        </EuiFlexItem>
-        {isAlert && !isPreview && (
-          <EuiFlexItem grow={false}>
-            <Assignees
-              eventId={eventId}
-              assignedUserIds={alertAssignees}
-              onAssigneesUpdated={onAssigneesUpdated}
-            />
-          </EuiFlexItem>
-        )}
-      </EuiFlexGroup>
+      {isAlert && (
+        <EuiPanel
+          hasShadow={false}
+          hasBorder
+          css={css`
+            padding: ${euiTheme.size.m} ${euiTheme.size.s};
+          `}
+          data-test-subj={ALERT_SUMMARY_PANEL_TEST_ID}
+        >
+          <EuiFlexGroup direction="row" gutterSize="m" responsive={false}>
+            <EuiFlexItem
+              css={css`
+                border-right: ${euiTheme.border.thin};
+              `}
+            >
+              <DocumentStatus />
```

Comment:
I recall we are keeping empty tags to avoid changing up the design of boxes. In the case of rule preview, risk score is available and is shown, status and user assignment are not available (previously hidden) and are now shown in empty tags.

@ferenrigue could you confirm?